extensions/objects/models/objects.py

```diff
@@ -139,6 +139,26 @@ def normalize(cls, raw):
             cls._normalize_value(raw),
             cls.get_schema())
 
+    @classmethod
+    def normalize_value(cls, raw):
```

Comment:
This is quite confusing -- it is named similarly to _normalize_value() but is different from it? I think this needs to be clearer conceptually...